src/TAction.cpp

```diff
@@ -138,7 +138,7 @@ bool TAction::compileScript()
     mFuncName = QString("Action")+QString::number( mID );
     QString code = QString("function ")+ mFuncName + QString("()\n") + mScript + QString("\nend\n");
     QString error;
-    if( mpHost->mLuaInterpreter.compile( code, error ) )
+    if( mpHost->mLuaInterpreter.compile( code, error, QString("Button: ") + getName() ) )
```

Comment:
This would attempt to translate the name of the button itself as well, no? That's surely not desirable.